src/components/ContentList.tsx

```diff
@@ -1,19 +1,24 @@
-import { SimpleGrid, Box, Text } from '@chakra-ui/react';
 import { Link } from 'react-router-dom';
 import { ROUTES } from '../constants/routes';
+import { Content } from './MainPageContent';
+import { SimpleGrid, Box, Text } from '@chakra-ui/react';
 
-type contents = { title: string; _id: string }[];
+type Props = {
```

Comment:
```suggestion
type ContentListProps = {
```
이런식으로 `type` 이름을 변경하는 것이 더 직관적일 것 같아요!!